engineering/incident-management.md

```diff
@@ -0,0 +1,140 @@
+**Incident management** is the practice of responding to an unplanned event, user disruption, or service interruption and involves restoring the service to an operational or acceptable state. There are 2 general labels for incidents:
+
+- **Incident:** An unplanned interruption to a service or reduction in the service quality.
+- **Major incident:** An incident with significant business impact, requiring an immediate coordinated resolution.
+
+A **problem** or **issue** is a not-yet-known root cause behind one or more incidents.
+
+## Purpose
+
+In order to establish trust in our software or the delivery of our software, we need to be timely and transparent with how we manage incidents. Good incident management involves not only high reactivity to difficulties, but proper follow-ups to notify relevant parties of the severity of the issue, how we manage the problem, and ETA and steps to resolution of the issue.
+
+## Incident management
+
+The overall procedure for monitoring alerts and taking action includes determining an incident's possible impact and severity and identifying it as an issue and giving it a priority. If a problem condition cannot be found, we will reject it as a non-impacting event.
+
+### Reporting the problem
```

Comment:
Do we need to mention we need to frequently do reporting? As the business and the customer need to know how long the problem has been resolved.